src/features/Navigation/view/Navigation.jsx

```diff
@@ -0,0 +1,5 @@
+function Navigation() {
```

Comment:
We use navigation only in header and we can locate it inside a header feature in folder components